tests/foreman/installer/test_installer.py

```diff
@@ -1395,14 +1416,29 @@ def test_capsule_installation(sat_default_install, cap_ready_rhel, default_org):
     assert sat_default_install.api.Capsule().search(
         query={'search': f'name={cap_ready_rhel.hostname}'}
     )[0]
+
+    # no errors/failures in journald
+    result = cap_ready_rhel.execute(
+        r'journalctl -q --no-pager -b -p err -u "pulpcore" -u "foreman" -u "foreman-proxy" -u "httpd" -u "postgresql" -u "pulpcore-api" -u "pulpcore-worker"'
+    )
+    assert len(result.stdout) == 0
+    # no errors/failures /var/log/foreman-installer/satellite.log
     result = cap_ready_rhel.execute(
         r'grep "\[ERROR" --after-context=100 /var/log/foreman-installer/satellite.log'
     )
     assert len(result.stdout) == 0
+    # no errors/failures /var/log/foreman-installer/capsule.log
     result = cap_ready_rhel.execute(
         r'grep "\[ERROR" --after-context=100 /var/log/foreman-installer/capsule.log'
     )
     assert len(result.stdout) == 0
+    # no errors/failures in /var/log/httpd/*
+    result = cap_ready_rhel.execute(r'grep -iR "error" /var/log/httpd/*')
+    assert len(result.stdout) == 0
+    # no errors/failures in /var/log/foreman-proxy/*
+    result = cap_ready_rhel.execute(r'grep -iR "error" /var/log/foreman-proxy/*')
```

Comment:
It would also be great to have more context if an error occurs. I like the approach above with `--after-context`.
Can you please consider adding it to the newly added commands? Eventually, you could use `--context` that would give us a context of `n` lines above as well as below the match.